support/client/lib/vwf/view/kineticjs.js

```diff
@@ -2045,4 +2080,16 @@ define( [ "module", "vwf/view", "jquery", "vwf/utility", "vwf/utility/color", "v
             eventHandler( parameters );
         }
     }
+
+    function refreshHitGraphFromCache( kineticObj ) {
+        if ( kineticObj.getAttrs().hitGraphFromCache && kineticObj.isVisible() ) {
+            kineticObj.clearCache();
+            kineticObj.draw();
+            kineticObj.cache();
+            kineticObj.drawHitFromCache();
+            return true;
+        }
+        return false;
+    }
```

Comment:
You could define this on the `this.state` object (defined in the model driver) and then use it in both the model and the view drivers.